### ISSUE TITLE
Usage information in package documentation does not work

### ISSUE THREAD

**Issue by graue70** [sjmgarnier/viridisLite#18]
The documentation of `?viridis` contains the following line:
```
viridisMap(n = 256, alpha = 1, begin = 0, end = 1, direction = 1,
  option = "D")
```
However, the function `viridisMap()` is not exported by the package, so the code cannot be run except with `viridisLite:::viridisMap()` (using the triple colon).

Either the function should be exported or it shouldn't be mentioned in the package documentation.

**Comment by sjmgarnier** [comment]
@graue70 Which version are you using? `viridisMap` should be an exported function in 0.3 if I'm not mistaken. In any case, this should work with the latest update from the repository.

**Comment by graue70** [comment]
> @graue70 Which version are you using? `viridisMap` should be an exported function in 0.3 if I'm not mistaken. In any case, this should work with the latest update from the repository.

No, it does not work with 0.3.0.

It is indeed fixed in 0.4.0.

**Comment by sjmgarnier** [comment]
@graue70 Really strange. I can't reproduce this here. The code definitely exports viridisMap in 0.3 as well. Oh well, I'm glad 0.4 fixes the problem somehow.
